site/pages/create-disruption.tsx

```diff
@@ -1,7 +1,8 @@
-import { ReactElement } from "react";
-import { DisruptionsDateTimeInfo } from "../components/DisruptionDateTimeInfo";
+import React, { ReactElement } from "react";
+import { DisruptionInfo } from "interfaces";
+import { DisruptionsDateTimeInfo } from "components/DisruptionDateTimeInfo";
+import { MiscellaneousReason, PersonnelReason, EnvironmentReason, EquipmentReason } from "../../shared-ts/siriTypes";
```

Comment:
These can be imported from the workspace package rather than using the relative path: `@create-disruptions-data/shared-ts/siriTypes`, then we shouldn't need to enable the externalDir feature

Reply:
duplicate comment